tauri-app/src/lib/__tests__/executeWalletConnectOrder.test.ts

```
@@ -0,0 +1,157 @@
+      mockOrderAddCalldataFn,
+      mockEthersExecuteFn,
+      mockReportErrorToSentryFn,
+      mockFormatEthersTransactionErrorFn,
+      mockSuccessToastFn,
+      mockErrorToastFn,
+    );
+
+    expect(mockOrderAddCalldataFn).toHaveBeenCalledWith(mockDotrainText, mockDeployment);
+    expect(mockEthersExecuteFn).toHaveBeenCalledWith(
+      mockCalldata,
+      mockDeployment.order.orderbook!.address,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)